packages/web/integrations/axelar/types.ts

```diff
@@ -22,53 +22,82 @@ export interface AxelarBridgeConfig {
   };
 }
 
-/** See: https://docs.axelar.dev/dev/build/chain-names/mainnet
- *  See: https://docs.axelar.dev/dev/build/chain-names/testnet
- *  Testnet: https://axelartest-lcd.quickapi.com/axelar/nexus/v1beta1/chains?status=1
- */
+/** Intermediate source chain identifiers */
 export type SourceChain =
```

Comment:
Let's change the comment to "human-displayable global source chain identifiers"

Reply:
Done